index.html

```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+                        <button class="pj-card-action" type="button">See Project</button>
+                    </section>
+                </section>
+                <section class="pj-card pj-5">
+                    <section class="pj-thumbnail"></section>
+                    <section class="pj-card-body">
+                        <h2 class="pj-card-title">Multi-Post Stories Gain+Glory</h2>
+                        <ul class="pj-card-lists">
+                            <li class="pj-card-list-item">
+                                <span>css</span>
+                            </li>
+                            <li class="pj-card-list-item">
+                                <span>javascript</span>
+                            </li>
+                            <li class="pj-card-list-item">
+                                <span>html</span>
+                            </li>
+                        </ul>
+                        <button class="pj-card-action" type="button">See Project</button>
+                    </section>
+                </section>
+                <section class="pj-card pj-6">
+                    <section class="pj-thumbnail"></section>
+                    <section class="pj-card-body">
+                        <h2 class="pj-card-title">Multi-Post Stories Gain+Glory</h2>
+                        <ul class="pj-card-lists">
+                            <li class="pj-card-list-item">
+                                <span>css</span>
+                            </li>
+                            <li class="pj-card-list-item">
+                                <span>javascript</span>
+                            </li>
+                            <li class="pj-card-list-item">
+                                <span>html</span>
+                            </li>
+                        </ul>
+                        <button class="pj-card-action" type="button">See Project</button>
+                    </section>
+                </section>
+            </section>
+        </section>
+
+        <!--About Me-->
+        <section id="about-me">
+            <section class="about-text">
+                <h3 class="about-heading">About me</h3>
+                <img id="about-icon" src="assets/Mask_Group.svg" alt="Background design for about section">
+                <p class="about-desc"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <img id="about-bg" src="assets/aboutFrame.svg" alt="">
+                <button class="about-btn" type="button">Get My Resume</button>
+            </section>
+
+            <ul class="categories">
+                <li class="list-item">
+                    <img src="assets/icon-languages.svg" alt="Languages Icon">
+
+                    <section class="list-body">
+                        <h3 class="list-title">Languages</h3>
+
```

Comment:
- [ ] Kindly make sure you are using the correct `color` on all **the texts** like the one provided on the Figma template